java/ql/lib/ext/java.nio.file.spi.model.yml

```diff
@@ -0,0 +1,6 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["java.nio.file.spi", "FileSystemProvider", True, "checkAccess", "(Path,AccessMode[])", "", "Argument[0]", "read-file", "ai-generated"]
```

Comment:
This is similar to the boolean-returning methods (e.g. `exists`) that we don't consider dangerous enough to be sinks. It may be interesting from a CFG perspective, but that's beyond models as data intended use. So I'd remove this file.